deeplinkdispatch-processor/build.gradle

```diff
@@ -5,20 +5,29 @@ sourceCompatibility = 1.7
 apply plugin: 'com.bmuschko.nexus'
 apply plugin: 'checkstyle'
 
+sourceSets {
+  test {
+    java {
+      srcDir '../deeplinkdispatch/src/main/java' // In tests we need access to the runtime files from the dld library
```

Comment:
why not add a regular `testImplementation` dependency on that?

Reply:
The processor is java module, and the `deeplinkdispatch` is now an Android module. Gradle doesn't seem to allow a java module to depend on an Android module. I don't love copying the class, but it got the job done. If the runtime component gets any more complicated, we should think of a different solution.

Reply:
ahh good point, thats a limitation.
you can do something like this instead then:
```
testImplementation project(':deeplinkdispatch').sourceSets.main.output
```
it should be a bit less fragile

Reply:
I tried your snippet and  `testImplementation project(':deeplinkdispatch').android.sourceSets.main.output`

But seems like android projects don't package output the same way. If you have any ideas I can follow up.